src/dmd/root/bitarray.d

```diff
@@ -55,6 +63,11 @@ nothrow:
             btc(ptr, idx);
     }
 
+    bool opEquals(const ref BitArray b) const
```

Comment:
ddoc comment?

Reply:
I agree, but given that Walter is unlikely to address this, I think that it's better to either push the change directly or open another PR

Reply:
Being a standard function pattern, I didn't see a need.

Reply:
Ok, thanks for the clarification on the standards